src/docs/product/alerts/notifications/index.mdx

```diff
@@ -1,79 +1,63 @@
 ---
 
-Sentry users with owner permissions and above can change default notification settings and configure notifications according to a project's needs. Notifications are categorized as [Workflow](#workflow), [Deploy](#deploy), and [Quota](#quota).
+Sentry sends you notifications regarding workflow activities, [release deploys](/product/releases/), and [quota usage](/product/accounts/quotas/), as well as weekly reports. These notifications let you know about:
 
-## Workflow
+- [Workflow](#workflow-notifications): Activity involving user actions and state changes on issues. This includes activities such as issue resolution, assignment, comments, and regressions.
+- [Deploy](#deploy-notifications): When a release you have commits on is deployed.
+- [Quota](#quota-notifications): Approaching quotas, exceeded quotas, and spike protection.
+- [Weekly Reports](#weekly-reports): A summary of your organization's Sentry activity.
 
-Workflow emails are sent for [issue state](#issue-states) changes. Workflow relates to activity involving user actions and state changes on issues and alerts, such as resolution, assignment, comments, and regressions.
+You can manage these notifications in **User Settings > Notifications**.
 
-By default, Sentry subscribes you to issues based on signals:
+## Workflow Notifications
 
-- Subscription to the issue or all workflow emails in the project
-- Involvement in a commit related to the issue
-- Commenting on or bookmarking the issue
-- Mention of you or your team in the issue
-- If you or your team is assigned to the issue
-
-### Issue States
-
-An issue can be in one of the following three states:
+Sentry sends workflow notifications to let you know about [issue state](/product/issues/states-triage/) changes. Workflow relates to actions that help you manage your issues, such as changing an issue’s state or commenting on it. By default, Sentry sends these notifications by email to members who are subscribed to the issue (see below for how subscription is determined). Workflow notifications are sent for:
 
-**Unresolved**: The default state when an issue is added to the system.
+- **Issue Resolved**: When a new issue is spotted in your code, it's in the Unresolved state. The issue state changes to Resolved when a project team member resolves it, either by manually changing its state in [sentry.io](https://sentry.io)) or by submitting a fix, or because of the project’s auto-resolve feature (if configured).
+- **Regressions**: A regression happens when the state of an issue changes from Resolved back to Unresolved. An email is sent to all project team members.
+- **Comments**: When a team member adds a new comment in the “Activity” tab of the detail page for the issue.
+- **Assignment**: When an issue is assigned or unassigned.
+- **User Feedback**: When an issue has new user feedback. <!--TO DO add link here when the product side user feedback content is created-->
```

Comment:
the layout is off here - I think it's because the end of the comment `-->` doesn't have a preceding space.

Also, I think the user feedback link would be to this SDK content `<PlatformLink to="/configuration/user-feedback/">user feedback</PlatformLink>`